gsa/src/web/pages/results/row.js

```diff
@@ -43,6 +45,10 @@ import TableData from '../../components/table/data.js';
 
 import ResultDelta from './delta.js';
 
+const Hostname = glamorous.span({
+  fontSize: '10px',
```

Comment:
What's the reason behind this fontSize? should hostname be smaller then other columns in the row? it's better to avoid fixed font sizes normally.